src/adagrams.js

```diff
@@ -1,15 +1,159 @@
+const handSize = 10
+
+const letterPool = {
```

Comment:
While we use the `const` keyword to ensure the values cannot be reassigned, we should also use capital letters to convey to other devs the value is fixed.
```suggestion
const HAND_SIZE = 10

const LETTER_POOL = {
```